### PR TITLE
update phase in the status of storagesystem

### PR DESCRIPTION
**Pull request by iamniting**
Signed-off-by: Nitin Goyal <nigoyal@redhat.com>

Should be merged after #41

### CODE REVIEW

File: controllers/subscriptions.go [hunk 1]
```diff
@@ -83,3 +89,29 @@ func (r *StorageSystemReconciler) ensureSubscription(instance *odfv1alpha1.Stora
 
 	return nil
 }
+
+func (r *StorageSystemReconciler) isCsvReady(instance *odfv1alpha1.StorageSystem, logger logr.Logger) error {
+
+	var csvName string
+
+	if instance.Spec.Kind == odfv1alpha1.FlashSystemCluster {
```
Comment (jarrpa) [inline controllers/subscriptions.go:97]: Change this to generation function.

File: controllers/subscriptions.go [hunk 2]
```diff
@@ -83,3 +89,29 @@ func (r *StorageSystemReconciler) ensureSubscription(instance *odfv1alpha1.Stora
...
+		csvName = IbmSubscriptionStartingCSV
+	} else if instance.Spec.Kind == odfv1alpha1.StorageCluster {
```
Comment (jarrpa) [inline controllers/subscriptions.go:99]: Change this to generation function.

**Review by jarrpa — CHANGES_REQUESTED** [review]

File: controllers/subscriptions.go (hunk 1 above)
Comment (iamniting) [reply controllers/subscriptions.go]: Done

File: controllers/subscriptions.go (hunk 2 above)
Comment (iamniting) [reply controllers/subscriptions.go]: Done

**Review by jarrpa — APPROVED** [review]
/lgtm

**Pull request merged**